src/components/TransferOption.js

```diff
@@ -1,6 +1,6 @@
-import propTypes from '@dhis2/prop-types'
 import { Tooltip } from '@dhis2/ui'
 import cx from 'classnames'
+import propTypes from 'prop-types'
```

Comment:
```suggestion
import PropTypes from 'prop-types'
```

and also rename in `propTypes` below.